trade-finance-manager-ui/server/helpers/date.js

```diff
@@ -64,6 +64,13 @@ const getBusinessDayOfMonth = (dateInMonth, holidays, businessDay) => {
   return result;
 };
 
+/**
+ * Converts date with index-0 month value to numeric index-1 month
+ * @param {Date} date
+ * @returns {number}
```

Comment:
We could change this to now return the `OneIndexedMonth` type

Reply:
The problem is that with type
```
export type OneIndexedMonth = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
```
if I have this as the return type it gets upset as `number is not assignable to OneIndexedMonth`. Can probably be ignored here, but in typescript we'd have to do a load of casting to satisfy the type system.

I think I'm going to change the type to
```
export type OneIndexedMonth = number;
```
as the extra type wrangling isn't worth it here, but the alias still gives us some extra documentation without having to rely on a comment.

Reply:
That's a shame, but we do get enough of a benefit from the alias I think so I'm happy with that